pkg/dashboard/store/alert.go

```diff
@@ -19,6 +19,23 @@ func (db *Store) Alerts() ([]*model.Alert, error) {
 	return data, err
 }
 
+func (db *Store) AlertsInWeek() ([]*model.Alert, error) {
+	query := sql.Stmt(db.driver, sql.SelectAlerts)
+	data := []*model.Alert{}
+	weekAgo := time.Now().Add(-7 * time.Hour * 24).Unix()
```

Comment:
This produces non-reproducible results.

Make it so that the exact time range is set. Week 48, week 49, etc. Need a more robust date logic.